### ISSUE TITLE
[ia-cloud-cnct2] パレット上のノード表示

### ISSUE THREAD

**Issue by ryoichi-obara** [ia-cloud/node-red-contrib-ia-cloud-fds#70]
![error-palette](https://user-images.githubusercontent.com/2470145/131430163-841522e8-3336-4af2-864c-40e86c74d1a0.png)

2つのノードについての英語名が設定されていないため、
何のノードなのかわからない状態になっています。

何か名前を考える必要があると思います。

**Comment by hashisan-atBridge** [comment]
リソースのJSONファイルに、editor.paletteLabelのエントリーがないのが第１原因ですが、

paletteLabel: function() {
            return this._("editor.paletteLabel") || "ia-cloud CCS";
        },

のコードで、this._("editor.paletteLabel")はエントリーのない場合は、"" か undefind あたりが帰るのかと思っていましたが、エントリーキーそのものが帰ってくるのですね。そういえばプロパティ編集画面のリソース参照でもそうですね。
完全にBugです。

 return this._("editor.paletteLabel") !==  "editor.paletteLabel" ? this._("editor.paletteLabel") : "ia-cloud CCS";

としなければならないですね。ほとんど全てのノードがそうです。
とりあえず、リソースファイルにエントリーがあれば、動作上はOKですが、気持ち悪いですね。
どうしましょうか？

**Comment by ryoichi-obara** [comment]
```js
return this.("editor.paletteLabel") !== "editor.paletteLabel" ? this.("editor.paletteLabel") : "ia-cloud CCS";
```

は結構冗長なコードな気がしますね。
Node-REDの仕様であるラベルがない場合はキーをそのまま返すという、ちょっと違和感のある動作にも依存していますし。

やはりラベル自体はパレット表示以外についても当然ちゃんと用意すべきということで、
コードでカバーしなくて良いと思います。
そこで、 Pull #71 のように変更するのがいいかなと思いPR出しておきました。

**Comment by hashisan-atBridge** [comment]
`this.("editor.paletteLabel")`
は、false を返すことはないようなので、
`return this.("editor.paletteLabel") ; `
とコード側も修正するのが気持ち良いですが、とりあえずPRマージしました。

本件、Issueに残しますか？

**Comment by ryoichi-obara** [comment]
そうですね、`this.("editor.paletteLabel")` がfalseやundefinedを返すのが嬉しいのですが、
現状はそんなことないようですね。

```js
oneditprepare() {
    return this.("editor.paletteLabel");
}
```

コードにこのように書くのはデフォルトの挙動と変わりないので、記載しないほうが簡潔かと思います。

> 本件、Issueに残しますか？

この議論(経緯)は今現在Issueでやりとりしているので残っていると思いますが、
全体としてのパレット上ノードのi18n対応とoneditprepare()削除ということでIssueを起票するかどうか？ということであれば私の意見はyesです。
